### ISSUE TITLE
Provide HTTPS & HTTP at the same time

### ISSUE THREAD

**Issue by iamchanghyunpark** [sameersbn/docker-gitlab#934]
Is it possible to host gitlab on both HTTPS and HTTP?
It seems if I set `GITLAB_HTTPS` to `true`, `GITLAB_PORT` works as the HTTPS port, and the HTTP port does not work.
The 80 port is forwarded, however I can't connect via HTTP

**Comment by solidnerd** [comment]
Hey, 

if you want to speak both you need to add your on nginx config to (assets/runtime/configs/nginx)[https://github.com/sameersbn/docker-gitlab/tree/master/assets/runtime/config/nginx] . I would not recommend this. I would use HTTPS only because why do you want HTTP ?

**Comment by iamchanghyunpark** [comment]
@solidnerd I just wrote up a long comment on why I would want HTTP, but then I realized that I shouldn't need it... (Closing issue) 
My initial concerns were

* When the certificate expires (I use Let's encrypt so that happens every 3 months) I may have trouble logging in.
* If I renew the certificate I may have to redo the copying of certificates, and during this time I may have numerous restarts of the container which prevents users from using gitlab.
* and etc.

So I was concerned about downtime of gitlab, however accessing via HTTP won't be possible either as I'm concerned about the time the containers are booting.
